app/person-escort-record/controllers/framework-overview.js

```diff
@@ -1,9 +1,10 @@
 const presenters = require('../../../common/presenters')
 
 function frameworkOverview(req, res) {
-  const { originalUrl, framework, personEscortRecord = {} } = req
-  const { meta, profile } = personEscortRecord
-  const fullname = profile?.person?.fullname
+  const { originalUrl, framework, personEscortRecord = {}, move = {} } = req
+  const { profile: moveProfile } = move
+  const { meta, profile: perProfile } = personEscortRecord
+  const fullname = perProfile?.person?.fullname || moveProfile?.person?.fullname
```

Comment:
Why not
```js
const profile = perProfile || moveProfile
const fullname = profile.person.fullname
```
Why would there not a profile on the move?

Reply:
Yes, that does read a little neater.

Reply:
sorry, "Why would there not a profile on the move?" was a separate question rather than  a clarification of the code suggestion ;)

Reply:
It's less about the move and more about if its using `/person-escort-record/:recordId` instead of `/move/:moveId/person-escort-record`.

In the former there is no `req.move`, but instead `req.personEscortRecord`.

Reply:
This essentially allows us to access the PER on its own or as part of a move.